### ISSUE TITLE
Syntax Error in autoAnimations.js/specialCaseAnimations.js - A-A v0.3.81

### ISSUE THREAD

**Issue by buttonpushertv** [otigon/automated-jb2a-animations#112]
I'm getting a similar error to #108 in FVTT 0.8.7 & A-A v0.3.81 (node.js install), but at a slightly different line - autoAnimation.js line 295 & specialCaseAnimations.js line 472.

In my case, this is appearing after either of two macros I run create a chat message ("DeathReasonsPlus" - https://pastebin.com/LJ6JjKWV & "Reset Token" - https://pastebin.com/LiNXdFeh). Both of those create a chat message and the syntax error is happening because the concatenated chat message contents are being passed to autoAnimations.js/specialCaseAnimations.js as an argument (I think?).

What's odd is that this "Create Chat Message" macro from the FVTT Community Macros does not create the error message - https://pastebin.com/gJ8f5pAw

Seems to be a different issue than #108 and still occurs after an update to A-A v0.3.81.

![screenshot 1624240179](https://user-images.githubusercontent.com/24526709/122696780-e436b680-d211-11eb-8b12-6ab1e0452a35.png)

**Comment by buttonpushertv** [comment]
Additionally, this error occurs during active combat encounter or outside of active combat encounter.

Oh, and I think that screen grab was made with the previous version of A-A (v0.3.80). The error is now occurring on line 299 of autoAnimations.js.

**Comment by otigon** [comment]
It's like playing whack-a-mole with that darn error. I'll test it with those macros and see what I can fix

**Comment by buttonpushertv** [comment]
Sure thing. If you want to use them exactly as I do then here are some of the extra bits you may need (to use them unaltered):

1. The "DeathReasons" Roll Table (called by "DeathReasonsPlus" macro)-  https://pastebin.com/M0eqex7m
2. The "Remove Conditions" Macro (called by "Reset Token" macro) - https://pastebin.com/48Nhzh58
3. You need to have CUB activated with Enhanced Conditions
4. And you need to set an Active Effect on the 'Dead' that calls the 'DeathReasonsPlus" macro (it need the args from that invoke method to run properly).

I don't know if any of those extra interactions may be contributing to the issue. I guess you could disable the CUB stuff if you don't think it would be part of the problem, but I defer to your knowledge about this stuff. I know juuuust enough about JS to get myself waaay in over my head very quickly.

**Comment by dirusulixes** [comment]
@otigon In case it helps I'm consistently getting the same error with fvtt 0.8.7 and AA 0.3.85 if I have the MidiQOl and Long Rest HD modules on (plus their dependencies).
![image](https://user-images.githubusercontent.com/5693623/123272623-d3c85980-d4f9-11eb-8e00-d4210268291e.png)

when I click for short or long rest I get:
```
jquery.min.js:2 Uncaught (in promise) Error: Syntax error, unrecognized expression: Akra (Dragonborn Cleric) takes a long rest.
    at Function.se.error (jquery.min.js:2)
    at se.tokenize (jquery.min.js:2)
    at se.select (jquery.min.js:2)
    at Function.se [as find] (jquery.min.js:2)
    at S.fn.init.find (jquery.min.js:2)
    at new S.fn.init (jquery.min.js:2)
    at S (jquery.min.js:2)
    at new Dnd5Handler (dnd5-handler.js:4)
    at specialCaseAnimations (autoAnimations.js:472)
    at autoAnimations.js:299
    ```

**Comment by otigon** [comment]
This issue should be resolved as of 0.4.00 thanks to @irbian
